mimir/src/main/scala/quasar/mimir/Precog.scala

```
@@ -96,7 +98,7 @@ final class Precog private (dataDir0: File)
     }
     val gated = vfsStr.mergeHaltBoth(vfsShutdownSignal.discrete.noneTerminate.drain)
 
-    gated.run.unsafePerformAsync(_ => ())
+    Precog.startTask(gated.run).unsafePerformSync
```

The alternative is that we log the exception, release the lock and have a `Precog` object which is in an invalid state.  I'd rather release the lock while setting a flag that indicates an error, so we can raise an exception directly rather than hitting an ambiguous `NullPointerException` at some point in the future.